docs/conf.py

```diff
@@ -192,6 +191,23 @@ def current_version() -> str:
 
 smartquotes = False
 
+linkcheck_ignore = [
+    # Ignore all HTML links to local files
```

Comment:
Not sure if you meant to ignore all HTTP links here but the comment seems irrelevant to the regex
```suggestion
    # Ignore all HTTP links to local files
```

Reply:
@alonmr there are links to local .html files which are in raw html format (for instance [here](https://github.com/mlrun/mlrun/blob/f1763a902e4049cb5422278c2233031ea94c4bf8/docs/index.md#L25)), we want to ignore them as these file don't exist in documentation folder.

Reply:
ok but the regex in the line below is for http so I don't understand how to comment matches the regex

Reply:
@alonmr no, it excludes all the http/https links